go.mod

```diff
@@ -3,21 +3,35 @@ module github.com/prometheus-community/smartctl_exporter
 go 1.17
```

Comment:
go 1.18 ? Or do you prefer to upgrade in a different PR ?

Reply:
That is just the minimum Go required. The actual Go version is in the CI/promu config.

Reply:
yeah I know, I'm just following what is done in Prometheus. Upgrading minimum go required with latest - 1 go version